lib/richmond/scan_result.rb

```diff
@@ -1,16 +1,16 @@
 class ScanResult
+
   attr_accessor :input, :output
 
   def initialize
-    @input = key_array_hash 
+    @input  = key_array_hash 
```

Comment:
See, it's a really minor thing, the spacing... but it seems **bigger** when it's **smaller**, like this one space. :smile: